### PR TITLE
[10.x] Adds latitude and longitude validation rule

### PR DESCRIPTION
**Pull request by dipesh79**
I have added two new validation rule, "latitude" and "longitude".

As the name suggests, latitude validates the input which are +- 90 and similarly  +-180 for longitude.

I had worked on a project where latitude and longitudes were used for map purpose but i had to make a custom regex validation for that. So, i am creating this PR to use the same validation from laravel framework.

Thank You.

### CODE REVIEW

**Comment by Dipesh79** [comment]
Hello @nunomaduro i am using phpStorm IDE while working on this PR. And i have used PSR-2 style for code style and I am still failing the style test.

Can you please help me. I might have missed something.

Thank You.

**Comment by driesvints** [comment]
Thanks. Please undo all unrelated styling changes first.

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php [hunk 1]
```diff
@@ -2460,6 +2460,30 @@ public function validateUuid($attribute, $value)
...
+    /**
+     * Validate that an attribute is a valid latitude.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateLatitude($attribute, $value)
+    {
+        return preg_match('/^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?)$/', $value);
```
Comment (imahmood) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php:2472]:
```suggestion
        return preg_match('/^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?)$/', $value) === 1;
```

File: src/Illuminate/Validation/Concerns/ValidatesAttributes.php [hunk 2]
```diff
@@ -2460,6 +2460,30 @@ public function validateUuid($attribute, $value)
...
+    /**
+     * Validate that an attribute is a valid longitude.
+     *
+     * @param  string  $attribute
+     * @param  mixed  $value
+     * @return bool
+     */
+    public function validateLongitude($attribute, $value)
+    {
+        return preg_match('/^[-+]?([1-8]?\d(\.\d+)?|180(\.0+)?)$/', $value);
```
Comment (imahmood) [inline src/Illuminate/Validation/Concerns/ValidatesAttributes.php:2484]:
```suggestion
        return preg_match('/^[-+]?([1-8]?\d(\.\d+)?|180(\.0+)?)$/', $value) === 1;
```

**Comment by achyutkneupane** [comment]
I always used regex for this validation. Thanks, @Dipesh79 for the PR. I hope it gets accepted.

**Comment by taylorotwell** [comment]
Thanks for your pull request to Laravel!

Unfortunately, I'm going to delay merging this code for now. To preserve our ability to adequately maintain the framework, we need to be very careful regarding the amount of code we include. 

If possible, please consider releasing your code as a package so that the community can still take advantage of your contributions! 

If you feel absolutely certain that this code corrects a bug in the framework, please "**@**" mention me in a follow-up comment with further explanation so that GitHub will send me a notification of your response.

**Pull request closed without merging**